Sources/ContainerClient/Core/ClientProcess.swift

```diff
@@ -45,79 +45,66 @@ public protocol ClientProcess: Sendable {
 
 struct ClientProcessImpl: ClientProcess, Sendable {
     static let serviceIdentifier = "com.apple.container.apiserver"
+
+    /// ID of the process.
+    public var id: String {
+        processId ?? containerId
+    }
+
     /// Identifier of the container.
     public let containerId: String
 
-    private let client: SandboxClient
-
     /// Identifier of a process. That is running inside of a container.
     /// This field is nil if the process this objects refers to is the
     /// init process of the container.
     public let processId: String?
 
-    public var id: String {
-        processId ?? containerId
-    }
+    private let client: XPCClient
 
-    init(containerId: String, processId: String? = nil, client: SandboxClient) {
+    init(containerId: String, processId: String? = nil, client: XPCClient) {
```

Comment:
It'd be nice to have a more descriptive name than `client` throughout this type.

`clientContainerClient` is like fingernails^v^v^chalkboard though.

Reply:
xpcClient is a good idea